helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ClusterEventMonitor.java

```diff
@@ -31,6 +31,7 @@
 import org.apache.helix.monitoring.mbeans.dynamicMBeans.HistogramDynamicMetric;
 import org.apache.helix.monitoring.mbeans.dynamicMBeans.SimpleDynamicMetric;
 
+
```

Comment:
I think the template is one line between the imports and the code, no?

Reply:
It's giving it two lines on the latest master format file.

Reply:
I personally don't like 2 lines. It makes the code looks longer and not concise. I see @pkuwm did that change before. @pkuwm, is there a particular reason we want 2 lines instead of one?

And @narendly, please put the reformat changes to a separate PR.

Reply:
@jiajunwang Double checked. The 2 lines setting is consistent with LinkedIn's java coding convention.
I actually prefer 1 line, as well. If we agree on making it 1 line, let me change it. Could we confirm that 1 line is preferred? Thanks.

Reply:
@pkuwm yes, I prefer 1 line.